app/views/admin/users/_reports.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="crayons-card p-6">
+  <div>
+    <h2 class="d-inline">Recent Reports</h2>
+    <button type="button" data-toggle="collapse" data-target="#reactions-row" class="btn btn-secondary float-right">Toggle</button>
+  </div>
+  <div class="pt-6 collapse" id="reactions-row">
+    <% unless @related_reports.empty? %>
+      <% @related_reports.each do |report| %>
+        <a href="<%= admin_feedback_messages_path %>" class="list-group-item list-group-item-action action px-5 d-flex justify-content-between">
```

Comment:
Let's use `admin_report_path(report.id)` instead here, to link to the individual report. We technically could do `admin_feedback_message_path(report.id)` too but `/admin/reports/:id` is a bit more popular.

Reply:
Great suggestion!